src/style/Components/_Screen.scss

```diff
@@ -14,4 +14,26 @@
       scroll-snap-align: start;
     }
   }
+
+  &--ArticleScreen {
+    .Hero {
+      justify-content: space-between;
+
+      .Hero__title {
+        text-align: center;
+
+        .type-h2 {
+          font-size: rem(14px);
+          color: $white-color;
+          letter-spacing: 1.2px;
+        }
+      }
+    }
+  }
+
+  &--Landing {
+    .ScrollDownButton {
```

Comment:
Screen-specific code should be moved to own file inside the screens directory for styles.